### PR TITLE
Fixed Dutch postal code validation.

### PR DESCRIPTION
**Pull request by Cerbrus**
This resolves issue #73

- Disabled left-padding on Dutch postal codes.
- Dutch postal codes can not start with a `0`.
- Dutch postal codes can not contain `SA`, `SS` or `SD`.

I see this is a generated file, but it's origins are unclear to me. If this should be implemented elsewhere, I'm happy to add another pull request. :-)

### CODE REVIEW

**Comment by Cerbrus** [comment]
Thank you, @mmaruniak. I'll have a look at that.

**Comment by Cerbrus** [comment]
@mmaruniak: Build succeeded now, but I had to disable some tests that deal with predecessors. The method that generates a predecessor doesn't seem to be able to handle numeric values that don't wrap around to 0. I'm not sure how to fix that.

File: src/PostalCodes.UnitTests/CountrySpecificPostalCodes/NLPostalCodeManualTests.cs [hunk 1]
```diff
@@ -33,13 +33,13 @@ public void Successor_ValidInput_ReturnsCorrectPostalCode(string postalCode, str
...
-        [TestCase("0000 AA")]
-        [TestCase("0000")]
-        public void Predecessor_FirstInRange_ReturnsNull(string postalCode)
-        {
-            Assert.IsNull((new NLPostalCode(postalCode)).Predecessor);
-        }
+        //[Test]
+        //[TestCase("1000 AA")]
+        //[TestCase("1000")]
+        //public void Predecessor_FirstInRange_ReturnsNull(string postalCode)
```
Comment (mmaruniak) [inline src/PostalCodes.UnitTests/CountrySpecificPostalCodes/NLPostalCodeManualTests.cs]: This test you can remove as I'm afraid there's no support for a min/max value other than zeroes and nines (an `A`s and `Z`s)

File: src/PostalCodes.UnitTests/Generated/NLPostalCodeTests.gen.cs [hunk 2]
```diff
@@ -33,11 +33,11 @@ public void Successor_ValidInput_ReturnsCorrectPostalCode(string postalCode, str
             Assert.AreEqual(codeSuccessor.ToHumanReadableString(), code.Successor.ToHumanReadableString());
         }
         
-        [TestCase("0000AA")]
-        public void Predecessor_FirstInRange_ReturnsNull(string postalCode)
-        {
-            Assert.IsNull((new NLPostalCode(postalCode)).Predecessor);
-        }
+        //[TestCase("1000AA")]
```
Comment (mmaruniak) [inline src/PostalCodes.UnitTests/Generated/NLPostalCodeTests.gen.cs]: Remove as well please

File: src/PostalCodes.UnitTests/CountrySpecificPostalCodes/NLPostalCodeManualTests.cs [hunk 3]
```diff
@@ -33,13 +33,13 @@ public void Successor_ValidInput_ReturnsCorrectPostalCode(string postalCode, str
             Assert.AreEqual(postalCodeSuccessor, (new NLPostalCode(postalCode)).Successor.ToString());
         }
 
-        [Test]
-        [TestCase("0000 AA")]
```
Comment (mmaruniak) [inline src/PostalCodes.UnitTests/CountrySpecificPostalCodes/NLPostalCodeManualTests.cs]: Could you add these test cases to the tests with invalid postal codes? Also add the `SA|SD|SS` cases please

**Comment by Cerbrus** [comment]
Done, @mmaruniak. Thanks for the review!

File: src/countries/NL.json [hunk 4]
```diff
@@ -22,19 +19,15 @@
 
     "TestData" : {
         "Max" : [ "9999ZZ" ],
-        "Min" : [ "0000AA" ],
-        "Valid" : ["1235DF", "5983DH"],
-        "Invalid" : [ "12j4h", "k3j51l" ],
+        "Min" : [ ],
```
Comment (mmaruniak) [inline src/countries/NL.json]: Would it be possible to have the Min `1000AA` here as you had in the previous commits, if it's not breaking anything? That would be my last comment :)

Comment (Cerbrus) [reply src/countries/NL.json]: @mmaruniak No, that generates a test case where the predecessor of postal code `1000AA` is checked. That predecessor throws an error, though, as the predecessor builder generates `0999ZZ`, which is an invalid postal code.

That's why that test case had to be removed: https://github.com/Cimpress-MCP/PostalCodes.Net/pull/74#discussion_r334885250

**Comment by Cerbrus** [comment]
@mmaruniak: Thanks for the merge.

Do you happen to have some information on if / when a new version of the package can be released?

**Comment by mmaruniak** [comment]
Sorry @Cerbrus we have a bit of a situation here. The API keys to publish the artifact are expired so I didn't manage to release it. And we are in process of transferring the repository to new owners so I'll get in touch with the original owners and try to fix the API keys

**Comment by Cerbrus** [comment]
Oh wow, yea, I can imagine that taking some time. Thanks for the update!

**Comment by Cerbrus** [comment]
@mmaruniak Is this still on the roadmap somewhere?
I just came across the ticket for this, on my end :-)

**Comment by mmaruniak** [comment]
Right, sorry for not getting back to this sooner @Cerbrus . I am out for a week but I have transferred this repository to @pavelsavara . Pavel could you please try releasing this? I think we only need to trigger a build on the 2.1.0 release branch now

**Comment by pavelsavara** [comment]
We just released version 3 for .NET standard 2.0. Is that ok for you ? 
(we do not have .NET framework build pipeline right now) @Cerbrus

**Comment by Cerbrus** [comment]
Hi @pavelsavara

I had to update the target framework of our project to 4.6.1, but that had to happen some day, any way.
Other than that minor tweak on our side, the package works like a charm!

Thanks for the update (@mmaruniak as well)!

**Pull request merged**